internal/config/config.go

```diff
@@ -96,7 +96,7 @@ type PodInfo struct {
 type Secret struct {
 	FileName   string                 `yaml:"fileName,omitempty"`
 	SecretPath string                 `yaml:"secretPath,omitempty"`
-	SecretType string                 `yaml:"secretType,omitempty"`
+	SecretType string                 `yaml:"secretType,omitempty"` // Deprecated, will be ignored
```

Comment:
why deprecate the secretType?

Reply:
It's not used anywhere in the code anyway, and I wanted to add functionality to `describe item` and based on the item type, get the item's value (static secret, certificate, etc...) - like in the external-secrets and secrets-webhook code.